GS01SUMO/GS01SUMO_R00004_ingest.csv

```diff
@@ -0,0 +1,80 @@
+mi.dataset.driver.phsen_abcdef.imodem.phsen_abcdef_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/phsen2/phsen2*.DAT,GS01SUMO-RII11-02-PHSENE042,recovered_host,Available,
+#mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/PHSEN_snP0179/SAMI*.txt,GS01SUMO-RII11-02-PHSENE042,recovered_inst,Not Expected,No data available
+mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/cpm1/irid2shore/cpm_status*.txt,GS01SUMO-SBC11-00-CPMENG000,recovered_host,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/syslog/*syslog.log,GS01SUMO-SBD11-00-DCLENG000,recovered_host,Available,
+mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/mopak/*.mopak.log,GS01SUMO-SBD11-01-MOPAK0000,recovered_host,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/hyd1/*hyd1.log,GS01SUMO-SBD11-03-HYDGN0000,recovered_host,Available,
+mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/dosta1/*.dosta1.log,GS01SUMO-SBD11-04-DOSTAD000,recovered_host,Available,
+mi.dataset.driver.spkir_abj.dcl.spkir_abj_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/spkir1/*.spkir1.log,GS01SUMO-SBD11-05-SPKIRB000,recovered_host,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/metbk1/*.metbk1.log,GS01SUMO-SBD11-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.suna.suna_dcl_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/nutnr1/*.nutnr1.log,GS01SUMO-SBD11-08-NUTNRB000,recovered_host,Available,
+mi.dataset.driver.suna.suna_instrument_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/NUTNR_sn*/DAT/*.CSV,GS01SUMO-SBD11-08-NUTNRB000,recovered_inst,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl12/syslog/*syslog.log,GS01SUMO-SBD12-00-DCLENG000,recovered_host,Available,
+#mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl12/optaa1/*.optaa1.log,GS01SUMO-SBD12-01-OPTAAD000,recovered_host,Not Deployed,
+mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl12/flort1/*.flort1.log,GS01SUMO-SBD12-02-FLORTD000,recovered_host,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl12/hyd2/*hyd2.log,GS01SUMO-SBD12-03-HYDGN0000,recovered_host,Available,
+mi.dataset.driver.pco2a_a.dcl.pco2a_a_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl12/pco2a1/*.pco2a1.log,GS01SUMO-SBD12-04-PCO2AA000,recovered_host,Available,

```

Comment:
Do we not have a driver for PCO2A recovered instrument data.  Also, we deployed 2 PCO2A instruments on this deployment (GS01SUMO-SBD11-03-PCO2AA000).  Telemetered data from that instrument is in OOINet.

Reply:
Another good question. Again, I don't see any recovered_inst streams for pco2a in the system, and I checked a few EA ingestion csvs and don't ingest recovered_inst for this instrument either. 

Good catch about the second pco2a, I missed that here. I will add it and upload a new version.